### DISCUSSION TITLE
How to access the Context arguments in a native function

### DISCUSSION THREAD

**Discussion by ddramireddy** [microsoft/semantic-kernel#6718] [General]
Hi Team,

I am looking for a way to access the context arguments in a native function in Semantic kernel. I tried to name function parameters as same name of the key in context arguments. But, it didn't help.

Is there any option to access these context arguments inside native function?

**Comment by stephentoub** [comment]
What language are you using? Can you share an example of what you tried?

**Comment by ddramireddy** [comment]
Never mind! It worked. We had some customization while calling the model. As soon as I pass the required details in the context, Model is able to extract the arguments and pass it. Following is the syntax, that I had it in rompt.my prompt.
rompt
{{PluginName.FunctionName $argument1 userId=$userId}}
